### PR TITLE
Fixing bootstrap and bootswatch CDN URLs, Updating themes to 4.6.0

### PR DESCRIPTION
**Pull request by mwhesse**

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/BootstrapSettings.java [hunk 1]
```diff
@@ -57,8 +57,8 @@ public BootstrapSettings() {
     }
 
     @Override
-    public IBootstrapSettings setVersion(String version) {
-        this.version = version;
+    public IBootstrapSettings setBootstrapVersion(String version) {
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/BootstrapSettings.java]: We cannot break the API in minor releases. Please revert.

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java [hunk 2]
```diff
@@ -11,10 +11,15 @@
...
+    /**
+     * The version of Bootstrap for Webjars
+     */
+    String BOOTSTRAP_WEBJARS_VERSION = "4.6.0-1";
+    
     /**
      * The version of Bootstrap
      */
-    String VERSION = "4.6.0-1";
+    String BOOTSTRAP_VERSION = "4.6.0";
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java]: Please revert the constant renaming.

Comment (martin-g) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java]: Why do we have now two versions - plain and webjars ? Can we use just `4.6.0` ?

**Review by martin-g — CHANGES_REQUESTED** [review]

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java (hunk 2 above)
Comment (mwhesse) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java]: There was a fix release "4.6.0-1" for the bootstrap 4.6.0 webjar. The  BOOTSTRAP_WEBJARS_VERSION constant is used in the BootstrapJsReferenceTest

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/BootstrapSettings.java (hunk 1 above)
Comment (mwhesse) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/BootstrapSettings.java]: Done

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
Thank you, @mwhesse !

**Pull request merged**